### PR TITLE
Newborn Exam - expand to History activity

### PR DESCRIPTION
**Pull request by anvmn**
#542

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-11-22 14-10

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Backend/Measurement/Encoder.elm [hunk 1]
```diff
@@ -3969,6 +4002,76 @@ encodeDeliveryComplication complication =
...
+            DefectHypospadias ->
+                "hypospadias"
+
+            DefectInguinalHernia ->
+                "inguinal-hernia"
+
+            DefectMicrocephaly ->
+                "microcephaly"
+
+            DefectNueralTubes ->
```
Comment (mariano-dagostino) [inline client/src/elm/Backend/Measurement/Encoder.elm]: @anvmn Perhaps this is `Neural Tubes` instead?

File: client/src/elm/Translate.elm [hunk 2]
```diff
@@ -2803,11 +2808,108 @@ translationSet trans =
...
+                    , kinyarwanda = Nothing
+                    }
+
+                DefectMicrocephaly ->
+                    { english = "Microcephaly"
+                    , kinyarwanda = Nothing
+                    }
+
+                DefectNueralTubes ->
+                    { english = "Nueral Tubes Defects"
```
Comment (mariano-dagostino) [inline client/src/elm/Translate.elm]: I'm pretty sure this is `Neural Tubes Defects`

**Review by mariano-dagostino — CHANGES_REQUESTED** [review]
Search and replace `Nueral` with `Neural` and `nueral` with neural

The rest looks good @anvmn

**Pull request merged**